rs/types/types/src/crypto.rs

```diff
@@ -341,6 +341,7 @@ pub enum CryptoError {
         algorithm: AlgorithmId,
         public_key_bytes: Vec<u8>,
         sig_bytes: Vec<u8>,
+        msg_hash: Option<Vec<u8>>,
```

Comment:
Partially this, also partially for confidentiality reasons in case the message contains sensitive information. Additionally for ECDSA the crypto component IIRC never sees the actual message but just the hash.